frontend/src/apis/visualization/.swagger-codegen/VERSION

```diff
@@ -0,0 +1 @@
+2.4.1
```

Comment:
I don't know if it will matter, but the other APIs were generated using `2.3.1`
Have you been able to verify that these newly generated API files work properly?

Reply:
Noted, I will generate with `2.3.1` instead.